### PR TITLE
Increase roster check time for sip test

### PR DESCRIPTION
**Pull request by katayal**
SIp test is failing because roster updates are not coming in time. Increasing the roster check duration.

### CODE REVIEW

File: integration/js/pages/AppPage.js [hunk 1]
```diff
@@ -279,20 +279,23 @@ class AppPage {
...
+          console.log("Roster check completed successfully in : %d secs", (start - end)/1000);
           return true;
         }
       } catch (err) {
       }
-      await TestUtils.waitAround(10);
+      await TestUtils.waitAround(waitTimeMs);
       i++;
     }
+    console.log("Roster check failed in : %d secs", (start - end)/1000);
```
Comment (ltrung) [inline integration/js/pages/AppPage.js:298]: So if roster check fail, where do we populate end variable here?

Comment (katayal) [reply integration/js/pages/AppPage.js:298]: oh yeah fixing it

File: integration/js/pages/AppPage.js [hunk 2]
```diff
@@ -279,20 +279,23 @@ class AppPage {
     return await this.driver.findElement(elements.failedMeetingFlow).isDisplayed();
   }
 
-  async rosterCheck(numberOfParticipants) {
+  async rosterCheck(numberOfParticipants, checkCount = 10, waitTimeMs = 100) {
```
Comment (ltrung) [inline integration/js/pages/AppPage.js:282]: nit: Why dont we just pass RosterCheckConfig here?

File: integration/js/pages/AppPage.js [hunk 3]
```diff
@@ -279,20 +279,24 @@ class AppPage {
...
+          console.log("Roster check completed successfully in : %d secs", (start - end)/1000);
           return true;
         }
       } catch (err) {
       }
-      await TestUtils.waitAround(10);
+      await TestUtils.waitAround(waitTimeMs);
       i++;
     }
+    let end = performance.now();
```
Comment (ltrung) [inline integration/js/pages/AppPage.js:298]: Variable end declared twice in this function.

File: integration/js/SipCallTest.js [hunk 4]
```diff
@@ -34,7 +34,7 @@ class SipCallTest extends SdkBaseTest {
     const sipCallClient = new SipCallClient(this.payload.sipTestAssetPath, this.payload.resultPath);
     const sipCall = sipCallClient.call(this.payload.voiceConnectorId, this.sipUri);
 
-    await test_window.runCommands(async () => await await RosterCheck.executeStep(this, session, 2));
+    await test_window.runCommands(async () => await await RosterCheck.executeStep(this, session, 2, new RosterCheckConfig(60, 500)));
```
Comment (ltrung) [inline integration/js/SipCallTest.js:37]: Will 30 second be enough? I thought this could take 35s before the SIP attendee shows up.

**Pull request closed without merging**